lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -956,6 +953,40 @@ const VtDictionary& UsdMayaJobImportArgs::GetDefaultDictionary()
         VtDictionaryOver(site, &d, /*coerceToWeakerOpinionType*/ true);
     });
 
+    // Shading options default value is variable and depends on loaded plugins.
+    // Default priorities for searching for materials, as found in
+    //  lib\mayaUsd\commands\baseListShadingModesCommand.cpp:
+    //    - Specialized importers using registry based import.
+    //    - Specialized importers, non-registry based.
+    //    - UsdPreviewSurface importer.
+    //    - Display colors as last resort
+    std::vector<VtValue> shadingModes;
+    for (const auto& c : UsdMayaShadingModeRegistry::ListMaterialConversions()) {
+        if (c != UsdImagingTokens->UsdPreviewSurface) {
+            auto const& info = UsdMayaShadingModeRegistry::GetMaterialConversionInfo(c);
+            if (info.hasImporter) {
+                shadingModes.emplace_back(std::vector<VtValue> {
+                    VtValue(UsdMayaShadingModeTokens->useRegistry.GetString()),
+                    VtValue(c.GetString()) });
+            }
+        }
+    }
+    for (const auto& s : UsdMayaShadingModeRegistry::ListImporters()) {
+        if (s != UsdMayaShadingModeTokens->useRegistry
+            && s != UsdMayaShadingModeTokens->displayColor) {
+            shadingModes.emplace_back(std::vector<VtValue> {
+                VtValue(s.GetString()), VtValue(UsdMayaShadingModeTokens->none.GetString()) });
+        }
+    }
+    shadingModes.emplace_back(
+        std::vector<VtValue> { VtValue(UsdMayaShadingModeTokens->useRegistry.GetString()),
+                               VtValue(UsdImagingTokens->UsdPreviewSurface.GetString()) });
+    shadingModes.emplace_back(
+        std::vector<VtValue> { VtValue(UsdMayaShadingModeTokens->displayColor.GetString()),
+                               VtValue(UsdMayaShadingModeTokens->none.GetString()) });
+
+    d[UsdMayaJobImportArgsTokens->shadingMode] = { VtValue(shadingModes) };
```

Comment:
This value will be used if the userArgs passed-in on import do not contain a shadingMode, which is what we want since the Maya-stored default does not have an opinion on this.